src/scss/_helpers.scss

```diff
@@ -0,0 +1,19 @@
+@import "_fonts";
+
+@mixin fontStyle($primaryFont, $weight, $size, $spacing) {
+    font-family: $primaryFont;
+    font-weight: $weight;
+    font-size: $size;
+    letter-spacing: $spacing;
+}
+
+@mixin fontProperties($weight, $size, $letterSpacing) {
+    @include fontStyle($primaryFont, $weight, $size, $letterSpacing);
+}
+
+@mixin fontWithSize($weight, $mobileSize, $desktopSize, $letterSpacing) {
+    @include fontProperties($weight, $mobileSize, $letterSpacing);
+    @include small-screen  {
```

Comment:
A more appropriate name would reflect that it is setting styles for larger screens (large-screen, desktop-screen)